<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePlayerDataWgt</class>
 <widget class="QWidget" name="GamePlayerDataWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GamePlayerDataWgt</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>物品</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>金币：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_gold"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>金币：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_bankGold"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>包裹</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QTableWidget" name="tableWidget_item">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>银行</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTableWidget" name="tableWidget_bankItem">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>新建行</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>新建列</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>宠物</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>包裹</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QTableWidget" name="tableWidget_pet"/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>银行</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableWidget" name="tableWidget_bankPet"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
